src/incubator/TextField/types.ts

```diff
@@ -162,13 +162,15 @@ export interface InputProps
    */
   readonly?: boolean;
 }
+type MandatoryField = {showMandatoryIndication?: boolean}
```

Comment:
since this type contain a single property you might as well add it to the main TextFieldProp list directly